libraries/MassServiceModeling/MassServiceModeling/Elements/Process.cs

```diff
@@ -1,32 +1,26 @@
 ﻿using DistributionRandomizer.DelayRandomizers;
+using MassServiceModeling.Items;
 using MassServiceModeling.Printers;
-using MassServiceModeling.ItemsQueues;
+using MassServiceModeling.Statistics;
+using MassServiceModeling.SubProcesses;
+
 namespace MassServiceModeling.Elements;
 
 public class Process : Element
 {
-    // Statistics
-    public int Failure { get; private set; }
-    public double MeanQueueAllTime { get; private set; }
-    
-    // SubProcesses
-    public List<SubProcess> SubProcesses { get; } = new();
-    public int WorkingSubProcessesCount => SubProcesses.Count(s => s.IsWorking);
-    protected List<SubProcess> SubProcessesForOutAct => SubProcesses.Where(s => s.NextT <= CurrT && s.IsWorking).ToList();
-    private SubProcess FreeSubProcess => SubProcesses.First(s => !s.IsWorking);
-
-    // Queue
     public event Action? OnQueueChanged;
-    public int QueueLength => Queue.Length;
-    protected ItemsQueue Queue;
+    public new ProcessStatisticHelper StatisticHelper;
+    public ItemsQueue Queue;
+    public SubProcessesHelper SubProcesses;
 
     public Process(Randomizer randomizer, int subProcessCount = 1, string name = "", int maxQueue = int.MaxValue, String subProcessName = "") 
         : base(randomizer, name)
     {
-        for (int i = 0; i < subProcessCount; i++)
-            SubProcesses.Add(new SubProcess(this, i, subProcessName));
+        SubProcesses = new SubProcessesHelper(this);
```

Comment:
Helper should "help" with external functions, in here it is more like SubprocessContainer